### PR TITLE
Balancepass 0.5.9

### PR DESCRIPTION
**Pull request by buckly90**
**Brief Description of What This PR Does**
* Decreased overall production of photosynthesis and thermosynthesis
* Fixed rustycyanin processes being doubled for sake of consistency. (Does not change gameplay.)
* Slightly lowered pilus damage
* Occurrence of ammonia and phosphate has been increased
* Caves no longer have abnormally small amounts of iron

**Progress Checklist**

- [ ] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [x] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

File: src/general/MainMenu.tscn [hunk 1]
```diff
@@ -417,6 +417,7 @@ text = "BACK"
 
 [node name="OptionsMenu" parent="." instance=ExtResource( 3 )]
 visible = false
+DisplayPartNamesTogglePath = null
```
Comment (hhyyrylainen) [inline src/general/MainMenu.tscn:420]: This seems a bit suspicious change...

Comment (buckly90) [reply src/general/MainMenu.tscn:420]: Uh oh, I must have accidently commited godot's changes

**Review by Oliveriver — APPROVED** [review]
Played for a while with these changes. While on the whole I agree with them - I think upping amounts of phosphate and ammonia was a particularly good change - I have discovered one problem.

It's surprisingly hard to be an effective predator now. The engulfment change means there are fewer options for prey as so many of them evolve protective membranes, and the lower pilus damage means stabbing takes so long you end up losing compounds in the overall hunt. So I'm in favour of increasing pilus damage, probably past what it was.

Otherwise, all seems good.

**Comment by buckly90** [comment]
You might be right about that. I lowered the pilus damage a bit to help lessen the impact of what I like to call "critical strikes" when a pilus makes contact with a cell in just the right way to cause damage numerous times within a short time frame. It can sometimes kill a cell nigh instantly.

**Comment by buckly90** [comment]
A good point, I'll change it back for now, and won't a change it until we can introduce a damage delay or something

**Review by hhyyrylainen — APPROVED** [review]
If the gameplay changes are good I don't have anything left to comment on the technical side of things.

**Pull request merged**